test/helpers/get-deployed-contracts-for-test.ts

```diff
@@ -61,6 +62,7 @@ async function getDeployedContractsForTest(): Promise<AllDeployedContracts> {
     await executeUpdateMerkleDistributorRewardsParametersProposalForTest(deployedContracts);
   }
   await executeWindDownSafetyModuleProposalForTest(deployedContracts);
```

Comment:
As a heads up, when using a block number after 16069909 in the mainnet fork tests, you'll have to move the `executeWindDownSafetyModuleProposalForTest` into the above if statement.

Reply:
yep sounds good, was originally testing before this latest one but I just moved it up in the latest commit